cli/device.cpp

```diff
@@ -518,6 +518,17 @@ bool VulkanDevice::format_is_supported(VkFormat format, VkFormatFeatureFlags for
 	return (format_features & supported) == format_features;
 }
 
+bool VulkanDevice::descriptor_set_layout_is_supported(const VkDescriptorSetLayoutCreateInfo *info)
+{
+	if (is_null_device)
+		return true;
+
+	VkDescriptorSetLayoutSupport set_layout_support = { VK_STRUCTURE_TYPE_DESCRIPTOR_SET_LAYOUT_SUPPORT };
+
+	vkGetDescriptorSetLayoutSupport(device, info, &set_layout_support);
```

Comment:
Probably a little more safe to use the KHR one in case a 1.0 device is used. maintenance3 is required for mutable, not 1.1.

Reply:
Sounds good.